src/server/game/Spells/SpellInfo.cpp

```diff
@@ -3442,6 +3442,11 @@ bool _isPositiveEffectImpl(SpellInfo const* spellInfo, uint8 effIndex, std::unor
             // Starfall
             if (spellInfo->SpellFamilyFlags[2] == 0x00000100)
                 return false;
+            break;
+        case SPELLFAMILY_DEATHKNIGHT:
+            if (spellInfo->SpellFamilyFlags[2] == 0x00000040) // Ebon Plague
```

Comment:
maybe handle SPELL_AURA_LINKED like SPELL_AURA_PERIODIC_TRIGGER_SPELL?

Reply:
We can define SPELL_AURA_LINKED, and add addtional checks for non positive target in trigger/linked spell, but in this case its not work. Linked spell have 0 in basepoints, this make linked from ebon plague is positive and ebon plague - positive